networks.js

```diff
@@ -13,55 +13,68 @@ const SHARED_DON_PUBLIC_KEY =
 const npmCommand = process.env.npm_lifecycle_event
 const isTestEnvironment = npmCommand == "test" || npmCommand == "test:unit"
 
-// Set EVM private key (required)
+// Set EVM private keys (required)
 const PRIVATE_KEY = process.env.PRIVATE_KEY
+
+// TODO @dev - set this to run the accept.js task.
 const SECOND_PRIVATE_KEY = process.env.SECOND_PRIVATE_KEY
+
 if (!isTestEnvironment && !PRIVATE_KEY) {
   throw Error("Set the PRIVATE_KEY environment variable with your EVM wallet private key")
 }
 
-// TODO @zeuslawyer - update contract addresses for sepolia etc.
+const accounts = []
+if (PRIVATE_KEY !== undefined) {
+  accounts.push(PRIVATE_KEY)
+}
+if (SECOND_PRIVATE_KEY !== undefined) {
+  accounts.push(SECOND_PRIVATE_KEY)
+}
+
 const networks = {
   ethereumSepolia: {
     url: process.env.ETHEREUM_SEPOLIA_RPC_URL || "UNSET",
     gasPrice: undefined,
-    accounts: PRIVATE_KEY !== undefined ? [PRIVATE_KEY, SECOND_PRIVATE_KEY] : [],
+    accounts,
     verifyApiKey: process.env.ETHERSCAN_API_KEY || "UNSET",
     chainId: 11155111,
     confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
     nativeCurrencySymbol: "ETH",
     linkToken: "0x779877A7B0D9E8603169DdbD7836e478b4624789",
     linkPriceFeed: "0x42585eD362B3f1BCa95c640FdFf35Ef899212734",
-    functionsOracleProxy: "0x649a2C205BE7A3d5e99206CEEFF30c794f0E31EC",
-    functionsBillingRegistryProxy: "0x3c79f56407DCB9dc9b852D139a317246f43750Cc",
+    functionsRouter: "", // TODO @zeuslawyer
+    donId: "", // TODO @zeuslawyer
+    gatewayUrls: "", // TODO @zeuslawyer
     functionsPublicKey: SHARED_DON_PUBLIC_KEY,
```

Comment:
We can also entirely remove SHARED_DON_PUBLIC_KEY from this file. It is no longer ever needed locally as it will always be pulled from on-chain.